python/demo.py

```diff
@@ -0,0 +1,148 @@
+# -*- coding: UTF-8 -*-
+import argparse
+import matplotlib.pyplot as plt
+import numpy as np
+import pandas as pd
+from pytorch_transformers import BertTokenizer, BertForTokenClassification, BertConfig
+import streamlit as st
+import torch
+import yaml
+
+# Run using `streamlit run demo.py`
+
+# NOTE: Change CHK_PATH to your preferred model checkpoint path before running.
+# NOTE 2: Argparse currently breaks the demo if you try to pass args in the command
+# line. If you'd like to use the demo in Russian, change argparse_default_lang to 'ru'
+# below.
+#
+# Example path:
+# CHK_PATH = "../models/10.03.2019, 17.12.59/train_checkpoint_epoch_5.tar"
+CHK_PATH = "YOUR_CHECKPOINT_PATH_HERE"
```

Comment:
These should go in the config yaml file, be modified in there, and the configs should be loaded in to this file

Reply:
Ahh I had a feeling that was the right way to go. Thanks!